### ISSUE TITLE
Add new "#interaction" marker

### ISSUE THREAD

**Issue by kphunter** [BCIT-LTC/dewordify#56]
We haven't had a standard way to incorporate interactive learning elements until a few days ago.

The new `#interactive` marker allows content developers to request custom interactive learning elements. It follows the following Word syntax:

```
#interaction
Title
source: {someURL}
/interaction
```

And the desired resultant HTML is:

```
<figure class="interaction">
  <iframe src="{someURL}"></iframe>
<figcaption>Title</figcaption>
</figure>
```

This will provide basic conversion functionality that can be tweaked manually in the code review.

**Comment by kphunter** [comment]
I propose that the we have `#interaction` map to `<pre>` in MarkoutMap.json as a temporary measure to make the manual process of embedding the interaction easier for builders.

I'll open a pull request.

**Comment by kphunter** [comment]
closed by #59...
